grid/recruitment/get-stage-changes/maps/breezy-hr.suma

```diff
@@ -0,0 +1,133 @@
+profile = "recruitment/get-stage-changes@1.0"
+provider = "breezy-hr"
+
+map GetStageChanges {
+  // TODO: add this error code to profile or use 'BadRequest'?
```

Comment:
I would go with `BadRequest` error, because this is limitation of the provider. The client needs to know that JobId is required input before perform of the use case. We should provide the information somewhere. It is a good question for the authoring feedback.

Reply:
True 👍🏻